### DISCUSSION TITLE
How to use dropdown with menu?

### DISCUSSION THREAD

**Discussion by vishal-ainow** [saadeghi/daisyui#3304] [General]
When a user clicks on the dropdown menu, it opens as shown in the screenshot. Is this possible?

Currently, it’s possible, but the menu is sticky and doesn’t cover the current view. Additionally, it doesn’t close by clicking anywhere or on dropdown button.

<img width="600" alt="Screenshot 2024-12-05 at 8 59 25 PM" src="https://github.com/user-attachments/assets/c7277d87-8afe-4dc8-b65a-dc7b7eab088c">

**Comment by vishal-ainow** [comment]
@saadeghi is it possible?

**Reply by saadeghi** [reply]
In dropdown, position is relative to the button, not the screen.  
If you need a section, elastic to the width of screen, I would suggest using JS to show/hide a div instead of using dropdown.

About the second question, it would be only possible using some JS,  
or with the new HTML "popover API" which will be available in daisyUI 5 soon.
